packages/core/app-layout/src/styles/variables/_app-layout.scss

```diff
@@ -1,2 +1,5 @@
 $app-layout-background: var(--kong-ui-app-layout-background, linear-gradient(180deg, #001740 0%, #073382 100%));
 $app-layout-main-box-shadow: var(--kong-ui-app-layout-main-box-shadow, -30px 174px 250px #0023db);
+// AccountDropdown
+$account-dropdown-background: var(--purple-300, #9396FC);
+$account-dropdown-color: var(--white, #fff);
```

Comment:
It looks like these two variables are no longer used now that you implemented the CSS variables; can you remove?